pkg/server/devfile-handler.go

```diff
@@ -50,33 +45,53 @@ func (s *Server) devfileHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	containerComponents := devfileObj.Data.GetDevfileContainerComponents() //TODO: filter thru Console attribute, right now if there is more than one component container err out
+	filterOptions := common.DevfileOptions{
+		Filter: map[string]interface{}{
+			"tool": "console-import",
+		},
+	}
+
+	containerComponents, err := devfileObj.Data.GetDevfileContainerComponents(filterOptions) //For Dev Preview, if there is more than one component container err out
+	if err != nil {
+		errMsg := fmt.Sprintf("Failed to get the container component from devfile with attribute 'tool: console-import': %v", err)
```

Comment:
@serenamarie125 This will require the owner of the Devfile is to add a couple lines to their Devfile in order for the UI to pick it up. This is so the Devfile can have multiple components but for Dev Preview we only deal with a singular targeted one. The error the user will see is in a red "danger" Alert of:

> Failed to get the container component from devfile with attribute 'tool: console-import'

Plus any error contents that may accompany the error from the library. I'll try to get a screenshot of this in a bit.